tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/fallback/AsyncFallbackClient.java

```diff
@@ -89,6 +89,7 @@
     @Asynchronous
     @Fallback(fallbackMethod = "fallbackCS")
     public CompletionStage<String> serviceCS3() {
+        RequestScoped rs;
```

Comment:
? :-)

Reply:
rs was not used at all. Not sure why it is there. It should be deleted.